frontend/src/components/deployments/api-deployment/ApiDeployment.jsx

```diff
@@ -275,6 +308,23 @@ function ApiDeployment() {
     },
     {
       key: "3",
+      label: (
+        <Space
+          direction="horizontal"
+          className="action-items"
+          onClick={() => downloadPostmanCollection()}
```

Comment:
@chandrasekharan-zipstack The usage of arrow function can be removed here. Can you please make the following change?

`onClick={downloadPostmanCollection}`